src/account/GoldEducation.tsx

```diff
@@ -26,7 +28,11 @@ export default function GoldEducation() {
     if (isCeloEducationComplete) {
       navigateBack()
     } else {
-      navigate(Screens.ExchangeHomeScreen)
+      getFeatureGate(StatsigFeatureGates.USE_TAB_NAVIGATOR)
```

Comment:
Wasn't able to write unit tests for this as I couldn't get the onFinish to be called by clicking the buttons 4 times. The Education component relies on scroll events to change the content which didn't seem to be called in the unit test. Open to other ideas for testing, but didn't want to spend too much time as this seems rarely used.